tuf/init.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/flynn/go-docopt"
+	"github.com/flynn/go-tuf"
+)
+
+func init() {
+	register("init", cmdInit, `
+usage: tuf init [--consistent-snapshot=false]
```

Comment:
The default will be true, the only way it will be false is if the flag is explicitly set to `false`